p2p/conn/secret_connection.go

```diff
@@ -42,6 +43,8 @@ const (
 	labelEphemeralUpperPublicKey = "EPHEMERAL_UPPER_PUBLIC_KEY"
 	labelDHSecret                = "DH_SECRET"
 	labelSecretConnectionMac     = "SECRET_CONNECTION_MAC"
+
+	defaultWriteBufferSize = 128 * 1024
```

Comment:
Re suggested buffer size: The maximum single message size is the encoding of a block part, which is 65kb. I set it to the size of two block parts, in case there is something else in the queue to be written

Reply:
The default buffer size for a writer is 4k:

https://cs.opensource.google/go/go/+/refs/tags/go1.22.4:src/bufio/bufio.go;l=19

I wonder whether is not better to the let the Writer to increase the buffer size on demand....

Reply:
The buffer's don't increase buffer size on demand though, after its too big, it just goes directly to the underlying writer (which is what we'd like to avoid!)